book/run/optimism.md

```diff
@@ -0,0 +1,118 @@
+> state is released, with the capability to extract the legacy state from the old `l2geth` LevelDB datadir and transplant it into Reth's MDBX database.
+
+You will need three things to run `op-reth`:
+1. An archival L1 node, synced to the settlement layer of the OP Stack chain you want to sync (e.g. `reth`, `geth`, `besu`, `nethermind`, etc.)
+1. A rollup node (e.g. `op-node`, `magi`, `hildr`, etc.)
+1. An instance of `op-reth`.
+
+### Installing `op-reth`
+
+To run Reth on Optimism, first install `op-reth` via the `Makefile` in the workspace root:
+
+```sh
+git clone git@github.com:paradigmxyz/reth.git && \
+    cd reth && \
+    make install-op
+```
+
+This will install the `op-reth` binary to `~/.cargo/bin/op-reth`.
+
+### Installing a Rollup Node
+
+Next, you'll need to install a [Rollup Node][rollup-node-spec], which is the equivalent to the Consensus Client on the OP Stack. Available options include:
+1. [`op-node`][op-node]
+1. [`magi`][magi]
+1. [`hildr`][hildr]
+
+For the sake of this tutorial, we'll use the reference implementation of the Rollup Node maintained by OP Labs, the `op-node`. The `op-node` can be built from source, or pulled from a [Docker image available on Google Cloud][op-node-docker].
+
+**`rethdb` build tag**  
+The `op-node` also comes with an experimental `rethdb` build tag, which allows it to read receipts directly from an L1 `reth` database during [derivation][derivation-spec]. This can speed up sync times, but it is not required if you do not
+have access to the L1 archive node on the same machine as your L2 node.
+
+To build the `op-node` with the `rethdb` build tag enabled:
+```sh
+git clone git@github.com:ethereum-optimism/optimism.git && \
+    (cd optimism/op-service/rethdb-reader && cargo build --release) && \ 
+    cd optimism/op-node && \
+    go build -v -tags rethdb -o ./bin/op-node ./cmd/main.go && \
+    mv bin/op-node /usr/bin/op-node
+```
+This will build the `rethdb-reader` dylib and instruct the `op-node` build to statically link this dylib into the binary. The `op-node` binary will be installed to `/usr/bin/op-node`.
+
+### Running `op-reth`
+
+The `optimism` feature flag in `op-reth` adds several new CLI flags to the `reth` binary:
+1. `--rollup.sequencer-http <uri>` - The sequencer endpoint to connect to. Transactions sent to the `op-reth` EL are also forwarded to this sequencer endpoint for inclusion, as the sequencer is the entity that builds blocks on OP Stack chains.
+1. `--rollup.disable-tx-pool-gossip` - Disables gossiping of transactions in the mempool to peers. This can be ommitted for personal nodes, though providers should always opt to enable this flag.
+1. `--rollup.enable-genesis-walkback` - Disables setting the forkchoice status to tip on startup, making the `op-node` walk back to genesis and verify the integrity of the chain before starting to sync. This can be ommitted unless a corruption of local chainstate is suspected.
+
+For this example, we'll start a `Base Mainnet` node.
```

Comment:
Let's put this at the top of the `## Running on Optimism` section so it's clear to people what they'll be running from the start.